rpcs3/Emu/RSX/Overlays/overlay_message_dialog.cpp

```diff
@@ -265,6 +266,8 @@ namespace rsx
 						}
 						return error;
 					}
+
+					g_last_user_response = return_code;
```

Comment:
How can you reproduce this?
Is there a specific dialog that surfaces this?
Or is this a regression from your changes?

Reply:
I could reproduce it pretty easily with save-dialog. Reverting the code changes hides the problem, but then if I spam x it sometimes does it (message dialog exits but flow reverts back to the save dialog)
It seems incorrect to me to have a blocking call that does not record the last user input and returns control flow to the caller.